src/DwellingAPI.php

```diff
@@ -72,4 +72,12 @@ public function loadStatuses() : Array
         $output = curl_exec($this->apiConnection);
         return json_decode($output);
     }
+
+    /**
+     * Get the value of apiUrl
+     */ 
+    public function getApiUrl()
+    {
+        return $this->apiUrl;
+    }
```

Comment:
This getter function is no longer being used anywhere and can be safely removed.